runelite-client/src/main/java/net/runelite/client/plugins/itemprices/ItemPricesOverlay.java

```diff
@@ -0,0 +1,262 @@
+			case ITEM_THIRD_OPTION:
+			case ITEM_FOURTH_OPTION:
+			case ITEM_FIFTH_OPTION:
+				// Item tooltip values
+				switch (groupId)
+				{
+					case WidgetID.INVENTORY_GROUP_ID:
+					case WidgetID.BANK_GROUP_ID:
+					case WidgetID.BANK_INVENTORY_GROUP_ID:
+						// Make tooltip
+						final String text = makeValueTooltip(menuEntry);
+						if (text != null)
+						{
+							tooltipManager.add(new Tooltip(text));
+						}
+						break;
+				}
+				break;
+		}
+		return null;
+	}
+
+	private String makeValueTooltip(MenuEntry menuEntry)
+	{
+		// Disabling both disables all value tooltips
+		if (!config.showGEPrice() && !config.showHAValue())
+		{
+			return null;
+		}
+
+		final int widgetId = menuEntry.getParam1();
+		// Inventory item
+		if (widgetId == INVENTORY_ITEM_WIDGETID || widgetId == BANK_INVENTORY_ITEM_WIDGETID)
+		{
+			final int index = menuEntry.getParam0();
+			// Find the item in the inventory to get stack size
+			final Item[] items = new InventoryItemQuery().result(client);
+			if (index < items.length)
+			{
+				final int id = items[index].getId();
+				final int qty = items[index].getQuantity();
+				return getItemStackValueText(id, qty);
+			}
+		}
+		// Bank item
+		else if (widgetId == BANK_ITEM_WIDGETID)
+		{
+			final int index = menuEntry.getParam0();
+			// Find the item in the bank to get stack size
+			final WidgetItem[] items = new BankItemQuery().result(client);
+			if (index < items.length)
+			{
+				final int id = items[index].getId();
+				final int qty = items[index].getQuantity();
+				return getItemStackValueText(id, qty);
+			}
+		}
+		return null;
+	}
+
+	private String getItemStackValueText(int id, int qty)
+	{
+		// Special case for coins and platinum tokens
+		if (id == ItemID.COINS_995)
+		{
+			return NUMBER_FORMATTER.format(qty) + " gp";
+		}
+		else if (id == ItemID.PLATINUM_TOKEN)
+		{
+			return NUMBER_FORMATTER.format(qty * 1000) + " gp";
+		}
+
+		final ItemComposition item = itemManager.getItemComposition(id);
+		// Only check prices for things with store prices
+		if (item.getPrice() > 0)
+		{
+			if (config.showGEPrice())
+			{
+				// Get grand exchange price async
+				// Will return null the first few times called for an item
+				final ItemPrice price = itemManager.getItemPriceAsync(id);
+				if (price != null)
+				{
+					if (config.showHAValue())
+					{
+						// Both grand exchange and high alch price
+						return stackValueText(qty, price.getPrice(), Math.round(item.getPrice() * HIGH_ALCHEMY_CONSTANT));
+					}
+					else
+					{
+						// Show only grand exchange price
+						return stackValueText(qty, price.getPrice(), 0);
+					}
+				}
+			}
+			else if (config.showHAValue())
+			{
+				// Show only high alch price
+				return stackValueText(qty, 0, Math.round(item.getPrice() * HIGH_ALCHEMY_CONSTANT));
+			}
+		}
+		return null;
+	}
+
+	private String stackValueText(int qty, int gePrice, int haValue)
+	{
+		itemStringBuilder.append("<col=eeeeee>");
+		if (qty <= 1)
```

Comment:
```java
			if (qty > 1 && (gePrice > 0 || haValue > 0))
			{
				itemStringBuilder.append("(").append(qty).append(") ");
			}
			if (gePrice > 0)
			{
				itemStringBuilder.append("EX: ")
						.append(StackFormatter.quantityToStackSize(gePrice * qty))
						.append(" gp");
				if (qty > 1 && config.showEA())
				{
					itemStringBuilder.append(" (")
							.append(StackFormatter.quantityToStackSize(gePrice))
							.append(" ea)");
				}
			}
			if (haValue > 0)
			{
				itemStringBuilder.append(gePrice > 0 ? ", " : "")
						.append("HA: ")
						.append(StackFormatter.quantityToStackSize(haValue * qty))
						.append(" gp");
				if (qty > 1 && config.showEA())
				{
					itemStringBuilder.append(" (")
							.append(StackFormatter.quantityToStackSize(haValue))
							.append(" ea)");
				}
			}
```

Reply:
Good catch, that's definitely unecessary